src/applications/vaos/services/appointment/index.js

```diff
@@ -677,3 +678,113 @@ export async function cancelAppointment({ appointment }) {
+  const isCommunityCare = appointment?.vaos.isCommunityCare;
+  const isPhone = isVAPhoneAppointment(appointment);
+  const isInPersonVAAppointment = !isVideo && !isCommunityCare && !isPhone;
+  const signinText = 'Sign in to VA.gov to get details about this appointment';
+
+  if (isPhone) {
+    data = {
+      summary: 'Phone appointment',
+      location: formatFacilityAddress(facility),
+      text: `A provider will call you at ${moment
+        .parseZone(appointment.start)
+        .format('h:mm a')}`,
+      phone: getFacilityPhone(facility),
+      additionalText: [signinText],
+    };
+  } else if (isInPersonVAAppointment) {
+    data = {
+      summary: `Appointment at ${facility?.name}`,
+      location: formatFacilityAddress(facility),
+      text: `You have a health care appointment at ${facility?.name}`,
+      phone: getFacilityPhone(facility),
+      additionalText: [signinText],
+    };
+  } else if (isCommunityCare) {
+    const { providerName, practiceName } =
+      appointment.communityCareProvider || {};
+
+    data = {
+      summary: `Appointment at ${providerName || practiceName}`,
+      providerName: `${providerName || practiceName}`,
+      location: `${formatFacilityAddress(appointment.communityCareProvider)}`,
+      text:
+        'You have a health care appointment with a community care provider. Please don’t go to your local VA health facility.',
+      phone: `${getFacilityPhone(appointment.communityCareProvider)}`,
+      additionalText: [signinText],
+    };
+  } else if (isVideo) {
+    const providerName = appointment.videoData?.providers
+      ? appointment.videoData.providers[0]?.display
+      : '';
+    const providerText = providerName
+      ? `You'll be meeting with ${providerName}`
+      : '';
+
+    if (isHome) {
+      data = {
+        summary: 'VA Video Connect appointment',
+        text:
+          'You can join this meeting up to 30 minutes before the start time.',
+        location: 'VA Video Connect at home',
+        additionalText: ['Sign in to VA.gov to join this meeting'],
+      };
+    } else if (isAtlas) {
+      const { atlasLocation } = appointment.videoData;
+
+      if (atlasLocation?.address) {
+        data = {
+          // TODO: This doesn't seem right
+          summary: `VA Video Connect appointment at an ATLAS facility`,
+          // providerName: `${firstName} ${lastName}`,
+          location: formatFacilityAddress(atlasLocation),
+          text: 'Join this video meeting from this ATLAS (non-VA) location:',
+          additionalText: [
+            `Your appointment code is ${
+              appointment.videoData.atlasConfirmationCode
+            }. Use this code to find your appointment on the computer at the ATLAS facility.`,
+            providerText,
+          ],
+        };
+      }
+    } else if (videoKind === VIDEO_TYPES.clinic) {
+      data = {
+        summary: `VA Video Connect appointment at ${
+          appointment.location.clinicName
```

Comment:
I think this should be the facility name, based on the copy doc. It also seems like it might need the facility name above the address in the description? https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/content/copy-docs/calendar-ics-copy.md#va-video-connect-at-va-location

This is how this type displays for me:
![Screen Shot 2021-05-19 at 3 43 56 PM](https://user-images.githubusercontent.com/634932/118874776-4326aa00-b8b9-11eb-864b-5bde240cc366.png)

Reply:
Done.